README-source.md

```diff
@@ -1211,7 +1211,29 @@ There are a lot of details left out - check out the full example app for a worki
 
 See [the wiki](https://github.com/zapier/zapier-platform-cli/wiki/Example-Apps) for a full list of working examples (and installation instructions).
 
-## Command line Tab Completion
+## FAQs
+
+*Q: Does Zapier support XML (SOAP) APIs?*
+
+A: Not natively, but it can! Users have reported that the following `npm` modules are compatible with the CLI Platform:
+
+* [pixl-xml](https://github.com/jhuckaby/pixl-xml)
+* [xml2js](https://github.com/Leonidas-from-XIV/node-xml2js)
+* [fast-xml-parser](https://github.com/NaturalIntelligence/fast-xml-parser)
+
+```javascript
+[insert-file:./snippets/xml.js]
+```
+
+*Q: Is it possible to iterate over pages in a polling trigger?*
+
+A: Yes, though there are caveats. Your entire function only gets 30 seconds to run. HTTP requests are costly, so paging through a list may timeout (which you should avoid at all costs). You should keep an the polling [guidelines](https://zapier.com/developer/documentation/v2/deduplication/), namely the part about only iterating until you hit items that weren't created in the last few hours.
```

Comment:
> "You should keep **an** the polling guidelines..."

I believe we have a typo here, but I don't know what it is.

Reply:
Should be "an eye". Thanks, I'll fix tomorrow!